tools/ipvsadm/ipvsadm.c

```diff
@@ -1810,6 +1864,9 @@ print_service_entry(ipvs_service_entry_t *se, unsigned int format)
 			left -= snprintf(svc_name + strlen(svc_name), left,
 				"MATCH %s", proto);
 
+			left -= snprintf(svc_name + strlen(svc_name), left,
+				",af=%s", af);
+
```

Comment:
"af" can be induced from the IP address in the "srange" and "drange". Do we need to display it? 
If "af" should be displayed here, I think it's better to place the string before "proto" and remove the preceding "af=" .

Reply:
What if there is no range.